backend/pkg/elasticsearch/augur_client.go

```diff
@@ -287,19 +288,37 @@ func ConvertToLogDocuments(data []map[string]interface{}) ([]logModel.LogEntry,
 	for _, item := range data {
 		doc := logModel.LogEntry{}
 
+		// Try multiple timestamp formats
+		layouts := []string{
+			"2006-01-02T15:04:05.000000000Z", // full format with nanoseconds
+			"2006-01-02T15:04:05.000Z",       // microseconds
+			"2006-01-02T15:04:05Z",           // seconds only
+		}
```

Comment:
When we use multiple layouts, time.Parse will attempt to match the timestamp against each layout. If one layout doesn't match, it tries the next one, which increases the chances of successfully parsing the timestamp.